### PR TITLE
Optional Style Change

### PR DESCRIPTION
**Pull request by pietrop**
## **Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

Not related to but in same domain as https://github.com/bbc/react-transcript-editor/issues/28

## **Describe what the PR does**    

I've been using this component in a couple of apps now, and I found it to be way too opinionated about the colour scheme it uses.
In more then one occasion it forced me to re style the surrounding parent component to avoid a design / UI  colour clash. And that is not great, coz Ideally I'd want it to be very quick to slot in for prototypes/proof of concepts etc.. And sometime a transparent background would be better suited.

But rather then suggest a colour scheme change, I've made some style customisation optional.

**TL;DR**: This PR that leaves current styles/colours as default, but allows to customise of the main elements of the UI.

## **State whether the PR is ready for review or whether it needs extra work**    

Ready for review

## **Additional context**    

- Does not introduce breaking changes
- added a guide in docs, and linked to it in README

Screenshot 2019-11-18 at 12 59 44

### Reviewing

if you want to view it in storybook, you can do it locally, or you can [check it out on my fork](http://pietropassarelli.com/react-transcript-editor/?path=/story/transcripteditor--video-file-custom-styles)  - http://pietropassarelli.com/react-transcript-editor

### CODE REVIEW

**Comment by pietrop** [comment]
**Examples**
Examples of situation where current styles is not ideal

Screenshot 2019-11-14 at 17 40 26

Screenshot 2019-10-30 at 15 39 14

Screenshot 2019-11-04 at 12 04 07

**Comment by pietrop** [comment]
Note: For some reason this branch was missing the latest speaker labels functionalities, and it wasn't showing up in the diffs, so just re-introduce those, ignore anything to do with TimedTextEditor and WrapperBlock when reviewing.

**Comment by pietrop** [comment]
closing this for now, there might be a better way to do this, but also not huge priority for now
